### PR TITLE
* Return user and sessionId on loginWithPassword

### PR DESCRIPTION
**Pull request by davidyaha**
* Fix email resolver on User type

### CODE REVIEW

File: packages/graphql-client/src/graphql-client.js [hunk 1]
```diff
@@ -42,6 +44,11 @@ export class GraphQLClient {
   }
 
   async loginWithPassword(user: string, password: string): Promise<LoginReturnType> {
+    const userFieldsFragment = this.options.userFieldsFragment ?
+      gql`${this.options.userFieldsFragment}` :
+      defaultUserFieldsFragment;
+    const loginMutation = createLoginMutation(userFieldsFragment);
+    console.log('Here!!!!!');
```
Comment (TimMikeladze) [inline packages/graphql-client/src/graphql-client.js]: Forgot to delete line?

File: packages/graphql-client/src/graphql/user-fileds.fragment.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import gql from 'graphql-tag';
+
+export const defaultUserFieldsFragment = gql`
+  fragment UserFields on User {
```
Comment (TimMikeladze) [inline packages/graphql-client/src/graphql/user-fileds.fragment.js]: what about the profile field?

**Review by TimMikeladze — APPROVED** [review]

File: packages/graphql-client/src/graphql/user-fileds.fragment.js (hunk 2 above)
Comment (davidyaha) [reply packages/graphql-client/src/graphql/user-fileds.fragment.js]: This is not in the graphql scheme but can be extended from the using code, check out scheme-generator

File: packages/graphql-client/src/graphql/index.js [hunk 3]
```diff
@@ -6,3 +6,4 @@ export * from './verify-email.mutation';
 export * from './reset-password.mutation';
 export * from './send-verification-email.mutation';
 export * from './send-reset-password-email.mutation';
+export * from './user-fileds.fragment';
```
Comment (pradel) [inline packages/graphql-client/src/graphql/index.js]: typo on the file name :)

**Review by dotansimha — APPROVED** [review]

**Pull request merged**